index.js

```diff
@@ -27,15 +27,36 @@ module.exports.onRendererWindow = async (window) => {
   }
 
   const config = window.config.getConfig();  
-  let deg = 0;
+  
+  if (config.hyperBorder && config.hyperBorder.animate) {
 
-  const updateBorderAngle = () => {
-    window.document.documentElement.style.setProperty('--border-angle', `${deg}deg`);
-    deg = deg > 360 ? 2 : deg + 2;
-  };
+    let deg = 0;
+    let intervalId = null;
```

Comment:
Should you use a module level variable rather than a local one? How does this behave when you modify the config at runtime, do we now have 2 intervals running?

Reply:
Okay, I checked this. The problem here is not that the intervals do not get cleared. This works. But we also have to clear the event listeners when the window is reloaded.